response_operations_ui/views/messages.py

```diff
@@ -176,6 +223,15 @@ def _refine(message):
     }
 
 
+def _get_survey_id(selected_survey):
+    survey_messages = survey_controllers.get_survey(selected_survey)
+    return survey_messages['survey']['id']
```

Comment:
We should add a method to the survey controller to get survey id by short name from the cache dict